test/server/cards/03_TWI/units/WolfPackEscort.spec.ts

```diff
@@ -0,0 +1,39 @@
+describe('Wolf Pack Escort', function () {
+    integration(function (contextRef) {
+        describe('ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['wolf-pack-escort'],
+                        groundArena: ['duchesss-champion', 'specforce-soldier'],
+                        spaceArena: ['star-wing-scout'],
+                    },
+                    autoSingleTarget: true
+                });
+            });
+
+            it('should return card to hand', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.wolfPackEscort);
+
+                // Checking we cannot select a vehicle
+                expect(context.player1).toBeAbleToSelectExactly([context.duchesssChampion, context.specforceSoldier]);
```

Comment:
We can also put `expect(context.player1).toHavePassAbilityButton()` to confirm that the optional part is set correctly